get-started/part4.md

```diff
@@ -415,8 +415,8 @@ From here you can do everything you learned about in parts 2 and 3.
 
 Scale the app by changing the `docker-compose.yml` file.
 
-Change the app behavior by editing code, rebuild, and push the new image.  (To
-do this, follow the same steps you took earlier to [build the
+Change the app behavior by editing code, rebuilding, and pushing the new image.
```

Comment:
I think the old way is actually more accurate (but poorly worded). You change the app behavior by editing code. Then you rebuild your image (with the edited code), and push the newly-rebuilt image.

Reply:
@mstanleyjones Right, that's why I had it worded that way originally but got comments to change it. I'll change it back.